src/components/Dropdown/_dropdown.scss

```diff
@@ -0,0 +1,82 @@
+@use '#scss/allPartials' as *;
+
+.#{$px}-dropdown {
+  @include text($body3);
+
+  &__trigger {
+    align-items: center;
+    background-color: $white;
+    border: none;
+    display: inline-flex;
+    gap: 10px;
+    height: 24px;
```

Comment:
I don't think we should have an explicit height. ideally the text size and styles along with the padding would equal the height we have in figma. Looking at figma I am also seeing a height of 32px.
<img width="121" alt="image" src="https://github.com/user-attachments/assets/8fd191b2-8513-4eb2-858a-d00b8d05cb7a">